AES.Entities/Tables/UserInfo.cs

```diff
@@ -31,6 +26,6 @@ public class UserInfo
         [RegularExpression(@"\d{3}-\d{3}-\d{4}")]
         public string Phone { get; set; }
 
-        public decimal SalaryExpectation { get; set; }
+        public double SalaryExpectation { get; set; }
```

Comment:
Nope, decimal is right. That's a C# special type for money. We don't need the precision a double gives.